style.css

```diff
@@ -1729,3 +1749,1416 @@ ul li {
   text-align: center;
   letter-spacing: 0.03em;
 }
+
+@media screen and (min-width: 850px) {
```

Comment:
- [ ] It is required that uses media queries for the Desktop breakpoint in the CSS file. (the value for min-width property should be in range of 768px or wider). This simply means that, the breakpoint for the desktop starts from 768px and above, not 850px and above. Kindly change `850px` to `768px`